backends-velox/src/main/scala/org/apache/gluten/extension/PartialProjectRule.scala

```diff
@@ -27,12 +27,12 @@ case class PartialProjectRule(spark: SparkSession) extends Rule[SparkPlan] {
     plan.transformUp {
       case plan: ProjectExec =>
         val transformer = ColumnarPartialProjectExec.create(plan)
-        if (transformer.doValidate().ok()) {
-          if (transformer.child.asInstanceOf[ColumnarPartialProjectExec].doValidate().ok()) {
-            transformer
-          } else plan
+        if (
+          transformer.doValidate().ok() &&
+          transformer.child.asInstanceOf[ColumnarPartialProjectExec].doValidate().ok()
+        ) {
+          transformer
```

Comment:
we should log the reason of validation failed.

Reply:
debug level?

Reply:
I think should be warning, same as default value of `spark.gluten.sql.validation.logLevel`

Reply:
`CollapseProjectExecTransformer` used debug level, then unify it into warning level?